release_notes.md

```diff
@@ -8,24 +8,8 @@
   - Fix thread context classloader for middleware chain
 - Update Java Worker Version to [2.15.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.15.0):
   - Update grpc-protobuf to 1.64.0 and application insights agent version to 3.5.2
-- Marking SyncTriggers [RequiresRunningHost]. (#10233)
-- Defaulting SwtAuthenticationEnabled to False (#10195)
-- Adding runtime site name to valid JWT audiences (slot scenarios) (#10185)
 - Update Python Worker Version to [4.30.3](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.30.3)
-- Language worker channels will not be started during placeholder mode if we are in-process (#10161)
 - Ordered invocations are now the default (#10201)
```

Comment:
"Ordered invocations are now the default" - Was this change already included in version 4.35.4?

Reply:
Yeap this is part of the release notes 4.35.4

Reply:
Shouldn't that be removed in that case , for 4.36.0 ?

Reply:
@kshyju  we removed this from release notes